src/lidar_subscriber.cpp

```diff
@@ -23,12 +23,6 @@ static bool same_landmark(const Observation2D &first_landmark, const Observation
 
 static void merge_landmarks(Observation2D &existing_landmark, Observation2D &new_measurement, int &count){
   existing_landmark.range_m = ((count-1)*existing_landmark.range_m+new_measurement.range_m)/count;
-  if(existing_landmark.bearing_rad>M_PI){
```

Comment:
This caused issues if the landmark was close to pi and some points were greater than pi and some were less than.